filament/src/Engine.cpp

```diff
@@ -94,29 +94,50 @@ FEngine* FEngine::create(Backend backend, Platform* platform, void* sharedGLCont
             instance->mOwnPlatform = true;
         }
         instance->mDriver = platform->createDriver(sharedGLContext);
-        instance->init();
         instance->execute();
```

Comment:
Is this `execute()` still needed since there's now one below after the `instance->init()`.

Reply:
oops no. good catch.